ui/apps/platform/src/services/schemas.js

```diff
@@ -1,5 +1,4 @@
 import { schema } from 'normalizr';
 
-export const image = new schema.Entity('image');
-
+/* eslint-disable import/prefer-default-export */
```

Comment:
I wonder if there is a reasonable way to configure this to cover only situations where we really want it. (On single component tsx files?) I get this warning a lot too when working on files that have a single export, but would make sense to have multiple in the future.

Reply:
That is a super suggestion!

Is *.tsx file the only situation where we really want it? Leaving JavaScript in the rear-view mirror ;(